packages/core/addon/templates/components/navi-visualizations/table.hbs

```diff
@@ -4,93 +4,23 @@
-              {{/if}}
-            </div>
-          {{/each}}
-        </div>
-      {{else}}
-        {{#sortable-group class='table-header-row' direction='x' onChange='updateColumnOrder' as | group |}}
-          {{#each columns as |column|}}
-            {{#sortable-item
-              class=(concat 'table-header-cell ' column.type)
-              model=(readonly column)
-              group=group
-              click=(action 'headerClicked' column)
-            }}
-                <span 
-                  class='table-header-cell__title {{if column.hasCustomDisplayName 'table-header-cell__title--custom-name'}}' 
-                  title='Drag column header to reorder'
-                >
-                    {{or column.displayName (default-column-name column)}}
-                </span>
-                {{#if column.sortDirection}}
-                  {{navi-table-sort-icon
-                    class='table-header-cell__icon'
-                    direction=(readonly column.sortDirection)
-                  }}
-                {{else}}
-                  {{tooltip-on-element
-                    text='Column cannot be sorted.'
-                    showOn="click"
-                    hideOn="mousedown mouseleave"
-                    duration=2000
-                    enableLazyRendering=true
-                  }}
-                {{/if}}
-            {{/sortable-item}}
-          {{/each}}
-        {{/sortable-group}}
-      {{/if}}
-
-    <div class='table-body'>
-      <div class='scroll-container'>
-        {{#table-iterator
-          tableData=tableData
-          rowDimensions=rowDimensions
-          occlusion=occlusion
-          as |row idx|
-        }}
-          <div class='table-row {{if row.__meta__.isTotalRow 'table-row__total-row'}} {{if (eq idx (sub tableData.length 1)) 'table-row__last-row'}}'>
-            {{#each columns as |column|}}
-              <div class="table-cell">
-                {{#if (and (eq column.type 'dateTime') row.__meta__.isTotalRow)}}
-                  {{component
-                    (concat cellRendererPrefix 'total')
-                      data=row
-                      column=column
-                  }}
-                {{else}}
-                  {{table-cell-renderer
-                      data=row
-                      column=column
-                      request=request
-                  }}
-                {{/if}}
-              </div>
-            {{/each}}
-          </div>
-        {{/table-iterator}}
-      </div>
-    </div>
-  </div>
-</div>
+{{component tableRenderer
+  tableData=tableData
+  request=request
+  columns=columns
+  occlusion=occlusion
+  estimateHeight=estimateHeight
+  bufferSize=bufferSize
+  isEditingMode=isEditingMode
+  cellRendererPrefix=cellRendererPrefix
+  headerClicked=(action 'headerClicked')
+  updateColumnOrder=(action 'updateColumnOrder')
+  updateColumnDisplayName=(action 'updateColumnDisplayName')
+  onUpdateReport=(action 'onUpdateReport')
```

Comment:
Can this be:
```hbs
onUpdateReport=(action onUpdateReport)
```

Reply:
I had tests failing in the dashboards package because we don't pass this action to the table component there
I've mentioned this bug on slack
once we fix that then sure, didn't want it to be part of this PR

Reply:
```js
onUpdateReport=(action optional (onUpdateReport))
```